dlt/helpers/airflow_helper.py

```diff
@@ -154,22 +154,28 @@ def add_run(
         Args:
             pipeline (Pipeline): An instance of pipeline used to run the source
             data (Any): Any data supported by `run` method of the pipeline
-            decompose (Literal["none", "serialize"], optional): A source decomposition strategy into Airflow tasks. Defaults to "none".
+            decompose (Literal["none", "serialize", "parallel"], optional):
+                A source decomposition strategy into Airflow tasks. Defaults to "none".
+                serialize - decompose the source into a sequence of Airflow tasks.
+                parallel - decompose the source into a parallel Airflow task group (NOTE:
```

Comment:
Please explain that we run first source compinent (ie. resource) and then all the rest in parallel

Reply:
Done